src/libsyntax/feature_gate.rs

```diff
@@ -519,6 +516,8 @@ declare_features! (
     // Allows the sysV64 ABI to be specified on all platforms
     // instead of just the platforms on which it is the C ABI
     (accepted, abi_sysv64, "1.24.0", Some(36167)),
+    // Allows a test to fail without failing the whole suite
+    (accepted, allow_fail, "1.23.0", Some(42219)),
```

Comment:
just a quick note: this should be ≥1.25.0 by now

Reply:
Oh indeed!